CRM/Core/Form/Task/PDFLetterCommon.php

```diff
@@ -30,7 +30,7 @@ class CRM_Core_Form_Task_PDFLetterCommon {
    * @var CRM_Core_Form $form
    */
   public static function preProcess(&$form) {
-    CRM_Utils_System::setTitle('Print/Merge Document');
+    $this->setTitle('Print/Merge Document');
```

Comment:
Dang this one is wrong - I was hoping I could skim this lightly to ensure all classes extend CRM_Core_Form & merge it a bit more 'lightly' with so much time ahead - but I wasn't looking for the `$this` vs `$form` pattern on my first pass from the bottom up to here so I just invalidated it :-(

Note that this function should have a deprecation notice in it as it is no longer called from core

![image](https://user-images.githubusercontent.com/336308/131910862-1abc644c-e064-4273-935a-7fdf40bb7f1b.png)

Reply:
@eileenmcnaughton Agh, missed this one. I think that's because I saw the PDFLetterCommon class and thought "I've seen you working on that one" then skipped it. The other clue here was that it's the only function that is static - now fixed.